os/net/lwip/configs/nd/Kconfig

```diff
@@ -3,7 +3,13 @@
 # see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
 #
 
-if NET_IPv6_ND
+menu "ND Support"
+
+config NET_IPv6_ND
+	depends on NET_IPv6
+	default y
+	---help---
+		Build in support for IPv6 Neighbor Discovery.
```

Comment:
Do you mean built** in support ?

Reply:
It's a mandatory option for IPv6 as described in this PR summary.